.github/dependabot.yml

```diff
@@ -48,3 +48,35 @@ updates:
       github_actions:
         patterns:
           - "*"
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      rust_dependencies:
+        patterns:
+          - "*"
+  - package-ecosystem: "cargo"
+    directory: "/xtask"
+    schedule:
+      interval: "weekly"
+    groups:
+      rust_dependencies:
+        patterns:
+          - "*"
```

Comment:
Would it be possible to only do a pr/update when a (cargo) semver break is done? e.g the first non-zero digit is changed. Reason is that I'm not sure we need to always be up to date, I can be persuaded about this though, I've just never seen the reason to keep track of the changes so closely.

Reply:
Maybe changing it to be done monthly?